Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.5
+
+MAINTAINER Zappi DevOps <devops@zappistore.com>
```

Comment:
Docker 1.13 dropped `MAINTAINER`
I think it should be replaced with:
`LABEL maintainer "Zappi DevOps <devops@zappistore.com>"`